Build error when compiling for Windows using MinGW

**Godot version:** https://github.com/godotengine/godot/commit/73cf0fd30524a6acc4400ef069210d90eaba29a2

**OS/device including version:** Fedora 28

**Issue description:** The build fails when compiling for Windows from Linux using MinGW. See the log attached below:

```
Compiling ==> drivers/wasapi/audio_driver_wasapi.cpp
drivers/wasapi/audio_driver_wasapi.cpp:35:10: fatal error: Functiondiscoverykeys_devpkey.h: No such file or directory
 #include <Functiondiscoverykeys_devpkey.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

[Full build log on GitLab CI](https://gitlab.com/Calinou/godot-builds-ci/-/jobs/88659560)

This was likely introduced by https://github.com/godotengine/godot/commit/f85062495c3ee477b7ab38f4de435435aedaf375 — it seems to be an [issue about case sensitivity](https://github.com/godotengine/godot/search?q=Functiondiscoverykeys) (notice the upper-case `F` in the path of the `#include` statement).

**Steps to reproduce:** Compile for Windows from Linux using MinGW.

--- a/drivers/wasapi/audio_driver_wasapi.cpp
+++ b/drivers/wasapi/audio_driver_wasapi.cpp
@@ -32,8 +32,6 @@
 
 #include "audio_driver_wasapi.h"
 
-#include <Functiondiscoverykeys_devpkey.h>
-
 #include "os/os.h"
 #include "project_settings.h"
 

